config/base/crds/kafka.banzaicloud.io_kafkaclusters.yaml

```diff
@@ -3491,6 +3491,18 @@ spec:
                     type: object
                   type: array
               type: object
+            disruptionBudget:
+              description: DisruptionBudget defines the configuration for PodDisruptionBudget
+              properties:
+                budget:
+                  description: The budget to set for the PDB, can either be static
+                    number or a percentage
+                  pattern: '[0-9]+(\.[0-9]+)?(%)?'
```

Comment:
How about this regex: `^[0-9]+$|^[0-9]{1,2}%$|^100%$` to not allow invalid percentage values such as 1000%